diff --git a/node_modules/@fortune-sheet/core/dist/index.esm.js b/node_modules/@fortune-sheet/core/dist/index.esm.js
index 5c9820b..c82f795 100644
--- a/node_modules/@fortune-sheet/core/dist/index.esm.js
+++ b/node_modules/@fortune-sheet/core/dist/index.esm.js
@@ -9789,11 +9789,11 @@ var en = {
   }],
   defaultFmt: function defaultFmt(currency) {
     return [{
-      text: "Automatic",
+      text: "자동",
       value: "General",
       example: ""
     }, {
-      text: "Plain text",
+      text: "일반 텍스트",
       value: "@",
       example: ""
     }, {
@@ -9801,15 +9801,15 @@ var en = {
       value: "split",
       example: ""
     }, {
-      text: "Number",
+      text: "숫자",
       value: "##0.00",
       example: "1000.12"
     }, {
-      text: "Percent",
+      text: "퍼센트",
       value: "#0.00%",
       example: "12.21%"
     }, {
-      text: "Scientific",
+      text: "과학적 표기법",
       value: "0.00E+00",
       example: "1.01E+5"
     }, {
@@ -9817,11 +9817,11 @@ var en = {
       value: "split",
       example: ""
     }, {
-      text: "Accounting",
+      text: "회계",
       value: "".concat(currency, "(0.00)"),
       example: "".concat(currency, "(1200.09)")
     }, {
-      text: "Currency",
+      text: "통화",
       value: "".concat(currency, "0.00"),
       example: "".concat(currency, "1200.09")
     }, {
@@ -9829,23 +9829,23 @@ var en = {
       value: "split",
       example: ""
     }, {
-      text: "Date",
+      text: "날짜",
       value: "yyyy-MM-dd",
       example: "2017-11-29"
     }, {
-      text: "Time",
+      text: "시간",
       value: "hh:mm AM/PM",
       example: "3:00 PM"
     }, {
-      text: "Time 24H",
+      text: "시간 (24H)",
       value: "hh:mm",
       example: "15:00"
     }, {
-      text: "Date time",
+      text: "날짜 및 시간",
       value: "yyyy-MM-dd hh:mm AM/PM",
       example: "2017-11-29 3:00 PM"
     }, {
-      text: "Date time 24 H",
+      text: "날짜 및 시간 (24H)",
       value: "yyyy-MM-dd hh:mm",
       example: "2017-11-29 15:00"
     }, {
@@ -9853,7 +9853,7 @@ var en = {
       value: "split",
       example: ""
     }, {
-      text: "Custom formats",
+      text: "맞춤 형식",
       value: "fmtOtherSelf",
       example: "",
       icon: "rightArrow"
@@ -9893,12 +9893,24 @@ var en = {
   fontFamily: {
     MicrosoftYaHei: "YaHei"
   },
-  fontarray: ["Times New Roman", "Arial", "Tahoma", "Verdana"],
+  fontarray: ["기본 글꼴", "맑은 고딕", "애플 SD 산돌고딕 Neo", "돋움", "굴림", "바탕체", "궁서체", "함초롬돋움", "함초롬바탕", "본고딕", "본명조", "프리텐다드", "나눔고딕", "나눔스퀘어 네오", "지마켓 산스", "배달의민족 주아"],
   fontjson: {
-    "times new roman": 0,
-    arial: 1,
-    tahoma: 2,
-    verdana: 3
+    "기본 글꼴": 0,
+    "맑은 고딕": 1,
+    "애플 sd 산돌고딕 neo": 2,
+    "돋움": 3,
+    "굴림": 4,
+    "바탕체": 5,
+    "궁서체": 6,
+    "함초롬돋움": 7,
+    "함초롬바탕": 8,
+    "본고딕": 9,
+    "본명조": 10,
+    "프리텐다드": 11,
+    "나눔고딕": 12,
+    "나눔스퀘어 네오": 13,
+    "지마켓 산스": 14,
+    "배달의민족 주아": 15
   },
   border: {
     borderTop: "Top border",
@@ -20479,30 +20491,24 @@ var zh = {
   fontFamily: {
     MicrosoftYaHei: "Microsoft YaHei"
   },
-  fontarray: ["Times New Roman", "Arial", "Tahoma", "Verdana", "微软雅黑", "宋体", "黑体", "楷体", "仿宋", "新宋体", "华文新魏", "华文行楷", "华文隶书"],
+  fontarray: ["기본 글꼴", "맑은 고딕", "애플 SD 산돌고딕 Neo", "돋움", "굴림", "바탕체", "궁서체", "함초롬돋움", "함초롬바탕", "본고딕", "본명조", "프리텐다드", "나눔고딕", "나눔스퀘어 네오", "지마켓 산스", "배달의민족 주아"],
   fontjson: {
-    "times new roman": 0,
-    arial: 1,
-    tahoma: 2,
-    verdana: 3,
-    微软雅黑: 4,
-    "microsoft yahei": 4,
-    宋体: 5,
-    simsun: 5,
-    黑体: 6,
-    simhei: 6,
-    楷体: 7,
-    kaiti: 7,
-    仿宋: 8,
-    fangsong: 8,
-    新宋体: 9,
-    nsimsun: 9,
-    华文新魏: 10,
-    stxinwei: 10,
-    华文行楷: 11,
-    stxingkai: 11,
-    华文隶书: 12,
-    stliti: 12
+    "기본 글꼴": 0,
+    "맑은 고딕": 1,
+    "애플 sd 산돌고딕 neo": 2,
+    "돋움": 3,
+    "굴림": 4,
+    "바탕체": 5,
+    "궁서체": 6,
+    "함초롬돋움": 7,
+    "함초롬바탕": 8,
+    "본고딕": 9,
+    "본명조": 10,
+    "프리텐다드": 11,
+    "나눔고딕": 12,
+    "나눔스퀘어 네오": 13,
+    "지마켓 산스": 14,
+    "배달의민족 주아": 15
   },
   border: {
     borderTop: "上框线",
@@ -30587,12 +30593,24 @@ var es = {
   fontFamily: {
     MicrosoftYaHei: "YaHei"
   },
-  fontarray: ["Times New Roman", "Arial", "Tahoma", "Verdana"],
+  fontarray: ["기본 글꼴", "맑은 고딕", "애플 SD 산돌고딕 Neo", "돋움", "굴림", "바탕체", "궁서체", "함초롬돋움", "함초롬바탕", "본고딕", "본명조", "프리텐다드", "나눔고딕", "나눔스퀘어 네오", "지마켓 산스", "배달의민족 주아"],
   fontjson: {
-    "times new roman": 0,
-    arial: 1,
-    tahoma: 2,
-    verdana: 3
+    "기본 글꼴": 0,
+    "맑은 고딕": 1,
+    "애플 sd 산돌고딕 neo": 2,
+    "돋움": 3,
+    "굴림": 4,
+    "바탕체": 5,
+    "궁서체": 6,
+    "함초롬돋움": 7,
+    "함초롬바탕": 8,
+    "본고딕": 9,
+    "본명조": 10,
+    "프리텐다드": 11,
+    "나눔고딕": 12,
+    "나눔스퀘어 네오": 13,
+    "지마켓 산스": 14,
+    "배달의민족 주아": 15
   },
   border: {
     borderTop: "borderTop",
@@ -41131,12 +41149,24 @@ var hi = {
   fontFamily: {
     MicrosoftYaHei: "YaHei"
   },
-  fontarray: ["Times New Roman", "Arial", "Tahoma", "Verdana"],
+  fontarray: ["기본 글꼴", "맑은 고딕", "애플 SD 산돌고딕 Neo", "돋움", "굴림", "바탕체", "궁서체", "함초롬돋움", "함초롬바탕", "본고딕", "본명조", "프리텐다드", "나눔고딕", "나눔스퀘어 네오", "지마켓 산스", "배달의민족 주아"],
   fontjson: {
-    "times new roman": 0,
-    arial: 1,
-    tahoma: 2,
-    verdana: 3
+    "기본 글꼴": 0,
+    "맑은 고딕": 1,
+    "애플 sd 산돌고딕 neo": 2,
+    "돋움": 3,
+    "굴림": 4,
+    "바탕체": 5,
+    "궁서체": 6,
+    "함초롬돋움": 7,
+    "함초롬바탕": 8,
+    "본고딕": 9,
+    "본명조": 10,
+    "프리텐다드": 11,
+    "나눔고딕": 12,
+    "나눔스퀘어 네오": 13,
+    "지마켓 산스": 14,
+    "배달의민족 주아": 15
   },
   border: {
     borderTop: "ऊपरी सीमा",
@@ -51238,30 +51268,24 @@ var zh_tw = {
   fontFamily: {
     MicrosoftYaHei: "Microsoft YaHei"
   },
-  fontarray: ["Times New Roman", "Arial", "Tahoma", "Verdana", "微軟雅黑", "宋體", "黑體", "楷體", "仿宋", "新宋體", "華文新魏", "華文行楷", "華文隸書"],
+  fontarray: ["기본 글꼴", "맑은 고딕", "애플 SD 산돌고딕 Neo", "돋움", "굴림", "바탕체", "궁서체", "함초롬돋움", "함초롬바탕", "본고딕", "본명조", "프리텐다드", "나눔고딕", "나눔스퀘어 네오", "지마켓 산스", "배달의민족 주아"],
   fontjson: {
-    "times new roman": 0,
-    arial: 1,
-    tahoma: 2,
-    verdana: 3,
-    微軟雅黑: 4,
-    "microsoft yahei": 4,
-    宋體: 5,
-    simsun: 5,
-    黑體: 6,
-    simhei: 6,
-    楷體: 7,
-    kaiti: 7,
-    仿宋: 8,
-    fangsong: 8,
-    新宋體: 9,
-    nsimsun: 9,
-    華文新魏: 10,
-    stxinwei: 10,
-    華文行楷: 11,
-    stxingkai: 11,
-    華文隸書: 12,
-    stliti: 12
+    "기본 글꼴": 0,
+    "맑은 고딕": 1,
+    "애플 sd 산돌고딕 neo": 2,
+    "돋움": 3,
+    "굴림": 4,
+    "바탕체": 5,
+    "궁서체": 6,
+    "함초롬돋움": 7,
+    "함초롬바탕": 8,
+    "본고딕": 9,
+    "본명조": 10,
+    "프리텐다드": 11,
+    "나눔고딕": 12,
+    "나눔스퀘어 네오": 13,
+    "지마켓 산스": 14,
+    "배달의민족 주아": 15
   },
   border: {
     borderTop: "上框線",
diff --git a/node_modules/@fortune-sheet/core/dist/index.js b/node_modules/@fortune-sheet/core/dist/index.js
index 04113a1..4c22649 100644
--- a/node_modules/@fortune-sheet/core/dist/index.js
+++ b/node_modules/@fortune-sheet/core/dist/index.js
@@ -9799,11 +9799,11 @@ var en = {
   }],
   defaultFmt: function defaultFmt(currency) {
     return [{
-      text: "Automatic",
+      text: "자동",
       value: "General",
       example: ""
     }, {
-      text: "Plain text",
+      text: "일반 텍스트",
       value: "@",
       example: ""
     }, {
@@ -9811,15 +9811,15 @@ var en = {
       value: "split",
       example: ""
     }, {
-      text: "Number",
+      text: "숫자",
       value: "##0.00",
       example: "1000.12"
     }, {
-      text: "Percent",
+      text: "퍼센트",
       value: "#0.00%",
       example: "12.21%"
     }, {
-      text: "Scientific",
+      text: "과학적 표기법",
       value: "0.00E+00",
       example: "1.01E+5"
     }, {
@@ -9827,11 +9827,11 @@ var en = {
       value: "split",
       example: ""
     }, {
-      text: "Accounting",
+      text: "회계",
       value: "".concat(currency, "(0.00)"),
       example: "".concat(currency, "(1200.09)")
     }, {
-      text: "Currency",
+      text: "통화",
       value: "".concat(currency, "0.00"),
       example: "".concat(currency, "1200.09")
     }, {
@@ -9839,23 +9839,23 @@ var en = {
       value: "split",
       example: ""
     }, {
-      text: "Date",
+      text: "날짜",
       value: "yyyy-MM-dd",
       example: "2017-11-29"
     }, {
-      text: "Time",
+      text: "시간",
       value: "hh:mm AM/PM",
       example: "3:00 PM"
     }, {
-      text: "Time 24H",
+      text: "시간 (24H)",
       value: "hh:mm",
       example: "15:00"
     }, {
-      text: "Date time",
+      text: "날짜 및 시간",
       value: "yyyy-MM-dd hh:mm AM/PM",
       example: "2017-11-29 3:00 PM"
     }, {
-      text: "Date time 24 H",
+      text: "날짜 및 시간 (24H)",
       value: "yyyy-MM-dd hh:mm",
       example: "2017-11-29 15:00"
     }, {
@@ -9863,7 +9863,7 @@ var en = {
       value: "split",
       example: ""
     }, {
-      text: "Custom formats",
+      text: "맞춤 형식",
       value: "fmtOtherSelf",
       example: "",
       icon: "rightArrow"
@@ -9903,12 +9903,24 @@ var en = {
   fontFamily: {
     MicrosoftYaHei: "YaHei"
   },
-  fontarray: ["Times New Roman", "Arial", "Tahoma", "Verdana"],
+  fontarray: ["기본 글꼴", "맑은 고딕", "애플 SD 산돌고딕 Neo", "돋움", "굴림", "바탕체", "궁서체", "함초롬돋움", "함초롬바탕", "본고딕", "본명조", "프리텐다드", "나눔고딕", "나눔스퀘어 네오", "지마켓 산스", "배달의민족 주아"],
   fontjson: {
-    "times new roman": 0,
-    arial: 1,
-    tahoma: 2,
-    verdana: 3
+    "기본 글꼴": 0,
+    "맑은 고딕": 1,
+    "애플 sd 산돌고딕 neo": 2,
+    "돋움": 3,
+    "굴림": 4,
+    "바탕체": 5,
+    "궁서체": 6,
+    "함초롬돋움": 7,
+    "함초롬바탕": 8,
+    "본고딕": 9,
+    "본명조": 10,
+    "프리텐다드": 11,
+    "나눔고딕": 12,
+    "나눔스퀘어 네오": 13,
+    "지마켓 산스": 14,
+    "배달의민족 주아": 15
   },
   border: {
     borderTop: "Top border",
@@ -20489,30 +20501,24 @@ var zh = {
   fontFamily: {
     MicrosoftYaHei: "Microsoft YaHei"
   },
-  fontarray: ["Times New Roman", "Arial", "Tahoma", "Verdana", "微软雅黑", "宋体", "黑体", "楷体", "仿宋", "新宋体", "华文新魏", "华文行楷", "华文隶书"],
+  fontarray: ["기본 글꼴", "맑은 고딕", "애플 SD 산돌고딕 Neo", "돋움", "굴림", "바탕체", "궁서체", "함초롬돋움", "함초롬바탕", "본고딕", "본명조", "프리텐다드", "나눔고딕", "나눔스퀘어 네오", "지마켓 산스", "배달의민족 주아"],
   fontjson: {
-    "times new roman": 0,
-    arial: 1,
-    tahoma: 2,
-    verdana: 3,
-    微软雅黑: 4,
-    "microsoft yahei": 4,
-    宋体: 5,
-    simsun: 5,
-    黑体: 6,
-    simhei: 6,
-    楷体: 7,
-    kaiti: 7,
-    仿宋: 8,
-    fangsong: 8,
-    新宋体: 9,
-    nsimsun: 9,
-    华文新魏: 10,
-    stxinwei: 10,
-    华文行楷: 11,
-    stxingkai: 11,
-    华文隶书: 12,
-    stliti: 12
+    "기본 글꼴": 0,
+    "맑은 고딕": 1,
+    "애플 sd 산돌고딕 neo": 2,
+    "돋움": 3,
+    "굴림": 4,
+    "바탕체": 5,
+    "궁서체": 6,
+    "함초롬돋움": 7,
+    "함초롬바탕": 8,
+    "본고딕": 9,
+    "본명조": 10,
+    "프리텐다드": 11,
+    "나눔고딕": 12,
+    "나눔스퀘어 네오": 13,
+    "지마켓 산스": 14,
+    "배달의민족 주아": 15
   },
   border: {
     borderTop: "上框线",
@@ -30597,12 +30603,24 @@ var es = {
   fontFamily: {
     MicrosoftYaHei: "YaHei"
   },
-  fontarray: ["Times New Roman", "Arial", "Tahoma", "Verdana"],
+  fontarray: ["기본 글꼴", "맑은 고딕", "애플 SD 산돌고딕 Neo", "돋움", "굴림", "바탕체", "궁서체", "함초롬돋움", "함초롬바탕", "본고딕", "본명조", "프리텐다드", "나눔고딕", "나눔스퀘어 네오", "지마켓 산스", "배달의민족 주아"],
   fontjson: {
-    "times new roman": 0,
-    arial: 1,
-    tahoma: 2,
-    verdana: 3
+    "기본 글꼴": 0,
+    "맑은 고딕": 1,
+    "애플 sd 산돌고딕 neo": 2,
+    "돋움": 3,
+    "굴림": 4,
+    "바탕체": 5,
+    "궁서체": 6,
+    "함초롬돋움": 7,
+    "함초롬바탕": 8,
+    "본고딕": 9,
+    "본명조": 10,
+    "프리텐다드": 11,
+    "나눔고딕": 12,
+    "나눔스퀘어 네오": 13,
+    "지마켓 산스": 14,
+    "배달의민족 주아": 15
   },
   border: {
     borderTop: "borderTop",
@@ -41141,12 +41159,24 @@ var hi = {
   fontFamily: {
     MicrosoftYaHei: "YaHei"
   },
-  fontarray: ["Times New Roman", "Arial", "Tahoma", "Verdana"],
+  fontarray: ["기본 글꼴", "맑은 고딕", "애플 SD 산돌고딕 Neo", "돋움", "굴림", "바탕체", "궁서체", "함초롬돋움", "함초롬바탕", "본고딕", "본명조", "프리텐다드", "나눔고딕", "나눔스퀘어 네오", "지마켓 산스", "배달의민족 주아"],
   fontjson: {
-    "times new roman": 0,
-    arial: 1,
-    tahoma: 2,
-    verdana: 3
+    "기본 글꼴": 0,
+    "맑은 고딕": 1,
+    "애플 sd 산돌고딕 neo": 2,
+    "돋움": 3,
+    "굴림": 4,
+    "바탕체": 5,
+    "궁서체": 6,
+    "함초롬돋움": 7,
+    "함초롬바탕": 8,
+    "본고딕": 9,
+    "본명조": 10,
+    "프리텐다드": 11,
+    "나눔고딕": 12,
+    "나눔스퀘어 네오": 13,
+    "지마켓 산스": 14,
+    "배달의민족 주아": 15
   },
   border: {
     borderTop: "ऊपरी सीमा",
@@ -51248,30 +51278,24 @@ var zh_tw = {
   fontFamily: {
     MicrosoftYaHei: "Microsoft YaHei"
   },
-  fontarray: ["Times New Roman", "Arial", "Tahoma", "Verdana", "微軟雅黑", "宋體", "黑體", "楷體", "仿宋", "新宋體", "華文新魏", "華文行楷", "華文隸書"],
+  fontarray: ["기본 글꼴", "맑은 고딕", "애플 SD 산돌고딕 Neo", "돋움", "굴림", "바탕체", "궁서체", "함초롬돋움", "함초롬바탕", "본고딕", "본명조", "프리텐다드", "나눔고딕", "나눔스퀘어 네오", "지마켓 산스", "배달의민족 주아"],
   fontjson: {
-    "times new roman": 0,
-    arial: 1,
-    tahoma: 2,
-    verdana: 3,
-    微軟雅黑: 4,
-    "microsoft yahei": 4,
-    宋體: 5,
-    simsun: 5,
-    黑體: 6,
-    simhei: 6,
-    楷體: 7,
-    kaiti: 7,
-    仿宋: 8,
-    fangsong: 8,
-    新宋體: 9,
-    nsimsun: 9,
-    華文新魏: 10,
-    stxinwei: 10,
-    華文行楷: 11,
-    stxingkai: 11,
-    華文隸書: 12,
-    stliti: 12
+    "기본 글꼴": 0,
+    "맑은 고딕": 1,
+    "애플 sd 산돌고딕 neo": 2,
+    "돋움": 3,
+    "굴림": 4,
+    "바탕체": 5,
+    "궁서체": 6,
+    "함초롬돋움": 7,
+    "함초롬바탕": 8,
+    "본고딕": 9,
+    "본명조": 10,
+    "프리텐다드": 11,
+    "나눔고딕": 12,
+    "나눔스퀘어 네오": 13,
+    "지마켓 산스": 14,
+    "배달의민족 주아": 15
   },
   border: {
     borderTop: "上框線",
